### PR TITLE
My branch

### PR DESCRIPTION
**Pull request by nadeemshare**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,2 +1,60 @@
...
-    </html>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <!-- <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css"
```
Comment (j-nadeem) [inline index.html:8]: tests

File: index.html
Comment (j-nadeem) [inline index.html:1]: testing comment

**Review by j-nadeem — APPROVED** [review]

File: index.html [hunk 2]
```diff
@@ -1,2 +1,60 @@
...
+          background-image: url('https://images.unsplash.com/photo-1603048675767-6e79ff5b8fc1?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80');
+        "
+      >
+        <h3>Explore the world</h3>
+      </div>
+      <div
+        class="panel"
+        style="
+          background-image: url('https://images.unsplash.com/photo-1595433502559-d8f05e6a1041?ixid=MXwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=1350&q=80');
+        "
```
Comment (j-nadeem) [inline index.html:53]: asdasdsd

**Review by j-nadeem — COMMENTED** [review]
d

**Comment by nadeemshare** [comment]
dsdfsdf

**Pull request merged**